src/content/pages/faq.mdx

```diff
@@ -23,11 +23,20 @@ subtitle: Frequently Asked Questions about EuroPython
 <li><b>2024-06-11</b>: [Schedule](/schedule) published</li>
 <li><b>2024-06-14</b>: [Remote Ticket Sales](/tickets#remote-tickets) & [Financial Aid for Remote Participation](/finaid#-free-remote-ticket) start</li>
 <li><b>2024-06-30</b>: [Sprints Venue & Info](/sprints) announced</li>
-</ul>
-
-<ul className="milestone-todo">
 <li><b>2024-07-08</b>: Together at EuroPython 2024! 🎊 </li>
-</ul>
+  <li><b>2024-07-13</b>: Sprint Weekend (13-14 July)</li>
+  <li><b>2024-07-14</b>: End of EuroPython 2024</li>
+  <li><b>2024-09-13</b>:  Celebrate the [Team of Europython 2024 ](/thank-you)</li>
+  <li><b>2024-09-16</b>: [EuroPython 2024 Photos](/photos) Published</li>
+  <li><b>2024-08-03</b>: [EuroPython 2024 Videos](https://www.youtube.com/playlist?list=PL8uoeex94UhE1CbtkDK4hevp2lBif57Nq) Published</li>
+  </ul>
+
+[//]: # ()
+[//]: # (<ul className="milestone-todo">)
+
+[//]: # (<li><b>2024-12-01</b>: Let's get ready for EuroPython 2025! 🎊 </li>)
```

Comment:
Aha is this deliberately commented out? Will this repo still be available for commits in Dec or might it have been archived and turned into static?

Reply:
Yes this is deliberate. I don't believe that anyone will ever uncomment this. But maybe this broad hint works. (You never know.)